numbers_letters_and_variables/hello_world.rb

```diff
@@ -1,3 +0,0 @@
-#!/usr/bin/env ruby
```

Comment:
Rebasing should have gotten rid of this...  Ask Wendy or I to help you fix it...